src/rules/use_isnan.rs

```diff
@@ -49,6 +49,43 @@ impl LintRule for UseIsNaN {
       ProgramRef::Script(ref s) => visitor.visit_script(s, &DUMMY_NODE),
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows comparisons to 'NaN'.
```

Comment:
```suggestion
    r#"Disallows comparisons to `NaN`.
```

Backticks would be nicer here because it will be rendered as a `<code>` tag so that readers can easily notice that it is a pre-defined value in JavaScript.